src/core/tests/frontend/paddle/test_models/gen_scripts/save_model.py

```diff
@@ -42,10 +42,31 @@ def print_array(arr, end=' '):
         print(print_array(arr, seperator_end))        
 
 
+def saveStaticModel(name, exe, feedlist:list, fetchlist:list, inputs:list, outputs:list, target_dir:str):
```

Comment:
Since saveModel and saveStaticModel are quite similar, can we merge them with an optional argument with default value?

Reply:
updated